components/Header.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-card flat tile>
+    <v-row>
+      <v-col cols="auto">
+        <v-img
+          src="https://cdn.vuetifyjs.com/images/cards/sunshine.jpg"
+          height="100"
+          width="150"
+        ></v-img>
+      </v-col>
+      <v-col cols="12" sm="8" md="8" lg="8" xl="8">
```

Comment:
```suggestion
<v-col cols="12" sm="8" >
```

`sm`と一緒なので `md="8" lg="8" xl="8"`はいらないはず

Reply:
@azure06 
`md="8" lg="8" xl="8"` 消しても動作に大差ないので消しちゃう対応pushしてマージします！